apps/application/models.py

```diff
@@ -90,6 +90,8 @@ class Application(models.Model):
     race_ethnicity = models.CharField(
         max_length=32, choices=RACE_ETHNICITY_CHOICES, null=True, blank=True)
 
+    priority = models.BooleanField(default=False)
```

Comment:
You should be able to avoid this field by using a `filter(created_at__lt=settings.PRIORITY_DATE)`. To order by this rule, you should be able to use Django annotations to create a variable (example [here](https://stackoverflow.com/a/32933336)).

Alternatively, it might be nice to keep the variable in case we ever want to assign priority status based on different criteria.

Reply:
Also, since priority status is only set during application creation, we would need to manually update all of the priority fields in this case.